lib/client-api/src/story_store.js

```diff
@@ -126,8 +95,8 @@ export default class StoryStore extends EventEmitter {
     );
   }
 
-  setSelection = data => {
-    this._selection = data;
+  setSelection = ({ storyId }) => {
+    this._selection = { storyId };
     setTimeout(() => this.emit(Events.STORY_RENDER), 1);
```

Comment:
If you're going to refactor this, I think we could remove the inheritance from EventEmitter and use the channel instead?

Just to have less moving parts and things to subscribe to?

Reply:
Probably yes. We weren’t sure of the full implications (eg in jest where there is no channel) so we held off doing that

Reply:
We'll do this later